TAScheduler/UnitTests/tests.py

```diff
@@ -2946,3 +2946,94 @@ def test_none_to_remove(self):
     def test_remove(self):
         self.lecture.removeTA()
         self.assertIsNone(self.lecture.getLectureTAAsgmt(), "removeTA() did not remove from lecture")
+
+class UserEditMyContactInfoTest(TestCase):
+    def setUp(self):
+        # Create a user instance for testing
+        self.user = User.objects.create(
+            email_address='test@example.com',
+            password='safe_password',
+            first_name='Test',
+            last_name='User',
+            home_address='123 Main St',
+            phone_number=1234567890
+        )
+
+    def test_edit_contact_info(self):
+        # Simulate the edit operation
+        self.user.first_name = 'UpdatedName'
+        self.user.save()
+
+        # Fetch the updated user from the database
+        updated_user = User.objects.get(id=self.user.id)
+
+        # Check if the first name is updated
+        self.assertEqual(updated_user.first_name, 'UpdatedName')
+
+
+class UserGetContactInfoTest(TestCase):
+    def setUp(self):
+        # Create a user instance for testing
+        self.user = User.objects.create(
+            email_address='test@example.com',
+            password='safe_password',
+            first_name='Test',
+            last_name='User',
+            home_address='123 Main St',
+            phone_number=1234567890
+        )
```

Comment:
If that's true you'll need to test from both TA, Instructor, and Admin

Reply:
Potentially you can just test one because it should work with everyone but Kiran is correct you can not do this primarly because you are calling the method on a database entry and I assume that these methods are going into views_methods.py so you need to call this on an object from that file, either AdminObj, TAObj, or InstructorObj